verticapy/tests/vModel/test_xgb_regressor.py

```diff
@@ -419,3 +413,29 @@ def test_get_plot(self, base, winequality_vd):
     def test_plot_tree(self, model):
         result = model.plot_tree()
         assert result.by_attr()[0:3] == "[1]"
+
+    def test_to_json(self, base, titanic_vd):
+        import xgboost as xgb
+
+        titanic = titanic_vd.copy()
+        titanic.fillna()
+        path = "verticapy_test_xgbr.json"
+        X = ["pclass", "age", "survived"]
+        y = "fare"
+        model = XGBoostRegressor("verticapy_xgb_regressor_test", max_ntree = 10, max_depth = 5, cursor = base.cursor)
+        model.drop()
+        model.fit(titanic, X, y)
+        X_test = titanic[X].to_numpy()
+        y_test_vertica = model.to_python()(X_test)
+        if os.path.exists(path):
+            os.remove(path)   
+        model.to_json(path)
+        model_python = xgb.XGBRegressor()
+        model_python.load_model(path)
+        y_test_python = model_python.predict(X_test)
+        result = abs(y_test_vertica - y_test_python) ** 2
```

Comment:
Calculating absolute before squaring is redundant.

Reply:
Done;